spec/chef/knife/inspect_spec.rb

```diff
@@ -15,6 +15,15 @@
       knife_inspect.run
     end
 
+    before do
```

Comment:
Something with Chef 11 is really weird, I had to do this to get the config hash to not be empty. Has anyone seen this before?